modules/userId/index.js

```diff
@@ -369,6 +369,9 @@ function initSubmodules(submodules, consentData) {
     utils.logWarn(`${MODULE_NAME} - gdpr permission not valid for local storage or cookies, exit module`);
     return [];
   }
+
+  const submodulesThatAlwaysRefresh = ['parrableId'];
```

Comment:
There should be an api config which states that parrableId needs refresh always. User id module should not hardcode id systems in the module. 

Please open a feature request/tech spec/separate PR for this feature. Prebid core would take a look and decide further.

Reply:
makes sense. I could not locate the api config to add this list? if you could point me where it is please, then I will abstract the feature into a new PR.